src/main/java/seedu/commands/CheckBudgetCommand.java

```diff
@@ -147,21 +146,22 @@ private void checkYear(ExpenditureList expenditures) throws EmptyStringException
                 BLANK);
         Year year = Year.parse(yearVal, formatYear);
         for (Expenditure individualExpenditure : expenditures.getExpenditures()) {
-            if (individualExpenditure.getDate().getYear() == year.getValue()) {
-                updateAmount(individualExpenditure);
-            }
-        }
-    }
-
-    private void checkMonth(ExpenditureList expenditures) throws EmptyStringException {
-        DateTimeFormatter formatMonth = DateTimeFormatter.ofPattern("uuuu-MM");
-        String monthVal = ParseIndividualValue.parseIndividualValue(userInput,
-                MSLASH, BLANK);
-        YearMonth yearMonth = YearMonth.parse(monthVal, formatMonth);
-        for (Expenditure individualExpenditure : expenditures.getExpenditures()) {
-            if (individualExpenditure.getDate().getYear() == yearMonth.getYear()
-                    && individualExpenditure.getDate().getMonth() == yearMonth.getMonth()) {
-                updateAmount(individualExpenditure);
+            if (individualExpenditure instanceof BorrowExpenditure) {
```

Comment:
Good thinking, borrowed and lent expenditure coincide should still reflect in the total expenditure count in between the set date and respective deadlines.